skillful_network_server/src/main/java/fr/uca/cdr/skillful_network/entities/user/User.java

```diff
@@ -99,6 +87,21 @@
 	@Transient
 	private Collection<? extends GrantedAuthority> authorities;
 
+	private FollowableStatus followableStatus = FollowableStatus.on;
+	private FollowableNotification followableNotifiable = FollowableNotification.all;
+
+	@OneToMany(mappedBy = "followed", cascade = {CascadeType.ALL}, fetch = FetchType.EAGER)
+//		@JoinColumn(name = "followed")
+//  	@JsonIgnoreProperties("user")
+//  	@JsonBackReference
+	Set<FollowStateTracker> followableSet = new HashSet<>();
+
+	@OneToMany(mappedBy = "follower", cascade = {CascadeType.ALL}, fetch = FetchType.EAGER)
+//		@JoinColumn(name = "follower")
+//		@JsonIgnoreProperties("user")
+//  	@JsonBackReference
+    Set<FollowStateTracker> followerSet = new HashSet<>();
```

Comment:
missing visibility

Reply:
not really needed . we used to implemented this way with Damien as a best practice